templates/pages/errors/404.html

```diff
@@ -1,14 +1,12 @@
 {{#partial "page"}}
 
-{{> components/common/breadcrumbs breadcrumbs=breadcrumbs}}
```

Comment:
@aleksandraworhacz same as with 403 page, please leave the breadcrumbs in the page. Thank you!

Reply:
I wonder why do we need breadcrumbs on error pages? They look a bit off IMHO

![Image 2023-09-08 at 1 02 17 PM](https://github.com/xfiveco/xcorner/assets/26577871/1ff260e5-bc36-41ea-bc74-25412a140f4e)

Reply:
@aleksandraworhacz hmm, you're right. Breadcrumbs here don't adds here. Thank you